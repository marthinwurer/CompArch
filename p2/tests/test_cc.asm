;
; tests SCC/CCC
;
; uses instructions HALT
;
; uses no addressing modes
;
	.globl	start
start:	scc
	ccc
	scc
	ccc
	halt
	.end	start
