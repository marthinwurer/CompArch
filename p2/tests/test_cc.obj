000 2 bf 00
002 2 af 00
004 2 bf 00
006 2 af 00
008 2 00 00
000
