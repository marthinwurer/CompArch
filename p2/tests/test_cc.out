CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 0
000000:  PS[00]  000277  SCC
000002:  PS[17]  000257  CCC
000004:  PS[00]  000277  SCC
000006:  PS[17]  000257  CCC
000010:  PS[00]  000000  HALT

Machine Halted - HALT instruction

Simulated time 21 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
